readme.md

```diff
@@ -90,6 +90,7 @@ aqt install-qt --outputdir "C:\Qt" windows desktop 6.5.0 win64_msvc2019_64 -m qt
 ```powershell
 mkdir C:\dev
 cd C:\dev
+bootstrap.bat
```

Comment:
This should be after the clone, and I don't think you can run it like that in powershell (at least with new powershell), you need `.\bootstrap.ps1`.

Reply:
Completely right, will get it updated

Reply:
Updated, I also pushed an update to the mob.ini